R/objectstore.R

```diff
@@ -101,7 +101,7 @@ save_object <- function(client,
     "GET",
     url,
     datasource_type = datasource$datasource_type,
-    write_disk = httr::write_disk(file),
+    write_disk = httr::write_disk(file, override = TRUE),
```

Comment:
@ddl-gabrielhaim did you test this? Because I think the parameter is `overwrite`, not `override`